packages/mobile/src/send/ValidateRecipientAccount.tsx

```diff
@@ -112,8 +112,10 @@ function navigateToConfirmationScreen(
     }
   } else {
     if (isOutgoingPaymentRequest) {
-      // This case should never happen yet.
-      navigateHome()
+      navigate(Screens.PaymentRequestConfirmation, {
+        transactionData: transactionData as TransactionDataInput,
+        addressJustValidated: true,
```

Comment:
This `addressJustValidated` is only used to call `Logger.showMessage(t('addressConfirmed'))` inside a `useEffect` hook. Can we instead delete this prop and just call the `showMessage` thing here instead? It feels like an unnecessary indirection to show the message in the React component. This `showMessage` thing shows a Toast btw, nothing related to rendering the UI.